src/test/java/flamegrapher/backend/JfrParserTest.java

```diff
@@ -12,6 +13,7 @@
 
 public class JfrParserTest {
 
+    @Ignore
```

Comment:
Can we remove it completely or add a .jfr as an example in test resources

Reply:
I was planning to do so (add the .jfr), just didn't have time yet.

Reply:
https://github.com/flamegrapher/flamegrapher/issues/8 
Added to handle this.